sw/airborne/modules/sonar/sonar_adc.c

```diff
@@ -20,31 +20,36 @@
  *
  */
 
-#include "modules/sonar/sonar_adc.h"
 #include "generated/airframe.h"
+#include "mcu_periph/uart.h"
+#include "pprzlink/messages.h"
+#include "subsystems/datalink/downlink.h"
+
+//#include "filters/median_filter.h"
+#include "filters/low_pass_filter.h"
+
 #include "mcu_periph/adc.h"
 #include "subsystems/abi.h"
 #ifdef SITL
 #include "state.h"
 #endif
+#include "modules/sonar/sonar_adc.h"
 
-#include "mcu_periph/uart.h"
-#include "pprzlink/messages.h"
-#include "subsystems/datalink/downlink.h"
-
-/** Sonar offset.
- *  Offset value in ADC
- *  equals to the ADC value so that height is zero
+/** Use low pass filter on pressure values
  */
-#ifndef SONAR_OFFSET
-#define SONAR_OFFSET 0
+#ifndef SONAR_USE_ADC_LOWPASS_FILTER
+#define SONAR_USE_ADC_LOWPASS_FILTER TRUE
```

Comment:
I think the default should be `FALSE` so the behavior doesn't change unexpectedly

Reply:
Yeas I thought of that, but with Analog sonar, and a feature that was never there, one we like to enable per default since that make the sensors usable in mast cases is the why it was chosen to set to TRUE per default